gradle/versions.gradle

```diff
@@ -133,6 +133,7 @@ dependencyManagement {
     dependency 'org.miracl.milagro.amcl:milagro-crypto-java:0.4.0'
     dependency 'org.quartz-scheduler:quartz:2.3.2'
     dependency 'org.rocksdb:rocksdbjni:6.11.4'
+    dependency 'org.fusesource.leveldbjni:leveldbjni-all:1.8'
```

Comment:
Oh, that's pretty old version (2014) :( 
The current native version is `1.22`: https://github.com/google/leveldb/releases/tag/1.22
`fusesource` had frozen on version `1.8` and we needed to release our own JNI artifact: https://bintray.com/ethereum/maven/EthereumJ-LevelDBJNI
But it's latest version `1.18.3` is also outdated :/

Reply:
Created #3587 to update LevelDB. Given its experimental still, probably makes sense to stick with the version from maven for now so we don't hit issues when bintray shuts down. Does look like we'll have to create our own fork and maintain the JNI wrapper ourselves.